content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
+jarang membutuhkan fork atau menambahkan patches ke kode projek Kubernetes.
+
+Panduan ini menjelaskan pilihan untuk menyesuaikan Kubernetes cluster.
+Ini ditujukan kepada {{< glossary_tooltip text="cluster operators" term_id="cluster-operator" >}} yang ingin
+memahami bagaimana menyesuaikan kluster Kubernetes dengan kebutuhan lingkungan kerja mereka.
+Pengembang yang prospektif {{< glossary_tooltip text="Platform Developers" term_id="platform-developer" >}} atau {{< glossary_tooltip text="Contributors" term_id="contributor" >}} Projek Kubernetes juga mendapatkan manfaat dari
+dokumen ini sebagai pengantar apa saja poin dan pola yang ada, untung-rugi dan batasannya.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Gambaran
+
+Pendekatan kostumisasi secara luas dapat dibagi atas *konfigurasi*, yang hanya melibatkan perubahan bendera, konfigurasi file lokal, atau API sumber;
+dan *ekstensi*, yang melibatkan berjalannya program atau layanan tambahan. Dokumen ini utamanya membahas tentang ekstensi.
+
+## Konfigurasi
+
+*File Konfigurasi* dan *bendera* terdokumentasi di bagian Referensi dokumentasi online, didalam setiap binary:
+
+* [kubelet](/docs/admin/kubelet/)
+* [kube-apiserver](/docs/admin/kube-apiserver/)
+* [kube-controller-manager](/docs/admin/kube-controller-manager/)
+* [kube-scheduler](/docs/admin/kube-scheduler/).
+
+Bendera dan file konfigurasi mungkin tidak selalu dapat diubah di layanan Kubernetes yang dihosting atau di distribusi dengan instalasi terkelola.
```

Comment:
```suggestion
_Flag-flag_ dan berkas-berkas konfigurasi mungkin tidak selalu dapat diubah pada layanan Kubernetes yang _hosted_  atau pada distribusi Kubernetes dengan instalasi yang dikelola.
```

Hosting is not an Indonesian word. Should we keep it as is (hosted) instead?

Reply:
okay, I'll keep it to 'hosted'

Reply:
done